### DISCUSSION TITLE
Surveying Team Proficiency in Programming Languages and Tools

### DISCUSSION THREAD

**Discussion by MiguelGibo** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#9] [Q&A]
To gauge the range of skills on our team, could everyone share which coding languages, markup languages, scripting languages, or other tools they are familiar with?

**Comment by AngelUPR** [comment]
I am most familiar with C++, Python, Java and html

**Comment by Mannyrodd101** [comment]
I have mostly experience on Python, Java, C++, HTML, CSS, and JavaScript.

**Comment by AlejandroClaudio** [comment]
I only have experience in Python, Java, and C++.

**Comment by pierreg-vales** [comment]
I have experience with python, java, c++, and a bit of HTML and CSS

**Comment by marymarmaldonado** [comment]
Like others have said too, I am most familiar with python, C++, Java and HTML/CSS!

**Comment by Kiarancole** [comment]
I have frontend experience with TypeScript, JavaScript and tools such as React and FluentUI. Also have experience with python, C++ and Java.

**Comment by estefania-2817** [comment]
I have experience with Python, Java, JavaScript, C++, HTML and CSS.

**Comment by LuisJCruz** [comment]
Python, Java, and C++

**Comment by Ojani** [comment]
I know html, css, javascript, Svelte (frontend framwork), nodeJS, expressJS, python, Dart, ElectronJs, a bit of java and c++, and a bit familiar with React

**Comment by KennethSepu1** [comment]
Python, Java, , C++, C#

**Comment by EdzelOrtiz** [comment]
Python, Javascrypt, CSS, HTML

**Comment by Ingrid1089** [comment]
I have experience with Python, Java and C++.
